packages/runtime/src/helpers/files.ts

```diff
@@ -397,26 +398,31 @@ const baseServerReplacements: Array<[string, string]> = [
     `checkIsOnDemandRevalidate(req, this.renderOpts.previewProps)`,
     `checkIsOnDemandRevalidate(process.env._REVALIDATE_SSG ? { headers: { 'x-prerender-revalidate': this.renderOpts.previewProps.previewModeId } } : req, this.renderOpts.previewProps)`,
   ],
+  // format of checkIsOnDemandRevalidate changed in 13.3.4
+  [
+    'checkIsOnDemandRevalidate)(req, this.renderOpts.previewProps)',
+    'checkIsOnDemandRevalidate)(process.env._REVALIDATE_SSG ? { headers: { "x-prerender-revalidate": this.renderOpts.previewProps.previewModeId } } : req, this.renderOpts.previewProps)',
+  ],
   // ensure ISR 404 pages send the correct SWR cache headers
   [`private: isPreviewMode || is404Page && cachedData`, `private: isPreviewMode && cachedData`],
 ]
 
 const nextServerReplacements: Array<[string, string]> = [
   [
     `getMiddlewareManifest() {\n        if (this.minimalMode) return null;`,
-    `getMiddlewareManifest() {\n        if (this.minimalMode || (process.env.NETLIFY && process.env.NEXT_DISABLE_NETLIFY_EDGE !== 'true' && process.env.NEXT_DISABLE_NETLIFY_EDGE !== '1')) return null;`,
+    `getMiddlewareManifest() {\n        if (this.minimalMode || (!process.env.NETLIFY_DEV && process.env.NEXT_DISABLE_NETLIFY_EDGE !== 'true' && process.env.NEXT_DISABLE_NETLIFY_EDGE !== '1')) return null;`,
```

Comment:
could do, but it will fail tests here until we get that one merged - in normal circumstances i'd be keen to split it out too, but for sake of getting this out the door i wonder whether we just keep it here? obvs feel free to split it out if you disagree 👍🏼